omnigan/losses.py

```diff
@@ -276,14 +276,16 @@ def __call__(self, prediction, target):
         relDiff = torch.mean(d) * torch.mean(d)
         return diff - relDiff
 
+
 class SIGMLoss(nn.Module):
-    def __init__(self, gmweight, device='cuda'):
+    def __init__(self, gmweight, device="cuda"):
```

Comment:
can you merge master into your branch before we merge yours ? this loss is outdated

Reply:
Done~